source/js/view.js

```diff
@@ -0,0 +1,220 @@
+
+    for (let i = 0; i < rows; i++) {
+        arr[i] = new Array(columns);
+    }
+    occupiedCells = arr;
+}
+
+function setListenerToCells() {
+    for (let i = 0; i < cells.length; i++) {
+        cells[i].addEventListener("click", function() {
+            let cellsRow = cells[i].id.split('-')[0];
+            let cellsCol = cells[i].id.split('-')[1];
+
+            viewModel.finishTurn(+cellsRow, +cellsCol);
+            updateCellContent();
+            changeCurrentStatus();
+        });
+    }
+}
+
+function updateCellContent() {
+    let fieldCellsArr = viewModel.getFieldCells();
+
+    for (let r = 0; r < fieldCellsArr.length; r++) {
+        for (let c = 0; c < fieldCellsArr[r].length; c++) {
+            if (fieldCellsArr[r][c] !== undefined
+                && fieldCellsArr[r][c] !== occupiedCells[r][c]) {
+                cells[r + '-' + c].innerHTML = fieldCellsArr[r][c];
+            }
+            occupiedCells[r][c] = fieldCellsArr[r][c];
+        }
+    }
+}
+
+function changeCurrentStatus() {
+    let statusField = document.getElementById("gameStatus");
+    statusField.innerHTML = viewModel.getPlayerName() + ' is ' + viewModel.getCurrentStatus();
+}
+
+/************ Building rows and columns ************/
+function increaseRows() {
+    rowsInput.value = viewModel.setRows(+rowsInput.value + 1);
+}
+
+function increaseColumns() {
+    columnsInput.value = viewModel.setColumns(+columnsInput.value + 1);
+}
+
+function increaseWinLength() {
+    winLengthInput.value = viewModel.setWinLength(+winLengthInput.value + 1);
+}
+
+function decreaseRows() {
+    rowsInput.value = viewModel.setRows(+rowsInput.value - 1);
+}
+
+function decreaseColumns() {
+    columnsInput.value = viewModel.setColumns(+columnsInput.value - 1);
+}
+
+function decreaseWinLength() {
+    winLengthInput.value = viewModel.setWinLength(+winLengthInput.value - 1);
+}
+
+/************ Building players ************/
+function updatePlayer(element, prop) {
+    let playerInstance = element.parentNode.parentNode;
+    let newVal = viewModel.updatePlayer(playerInstance.id, prop, element.value);
+    if (prop === 'name') {
+        playerInstance.querySelector('.form-group .player_name').value = newVal;
+    }
+    if (prop === 'symbol') {
+        updateSymbolList();
+    }
+}
+
+function addPlayer() {
+    let defaultType = 'human';
+    let defaultName = 'Player';
+    let defaultSymbol = availableSymbolList[0];
+    if (viewModel.addPlayer(defaultType, defaultName, defaultSymbol)) {
+        let list = viewModel.getPlayerList();
+        let newPlayer = list[list.length - 1];
+        getPlayerInstanceTemplate(newPlayer);
+        updateSymbolList();
+    }
+}
+
+function removePlayer(element) {
+    let playerToRemove = element.parentNode.parentNode;
+    let id = parseInt(playerToRemove.id, 10);
+
+    if (viewModel.removePlayer(id)) {
+        playerInstanceBlock.removeChild(playerToRemove);
+    }
+}
+
+function updateSymbolList() {
```

Comment:
Please remove this function and see the comment for the `createSymbolList` function